app/scripts/ui.js

```diff
@@ -117,11 +117,13 @@ async function start() {
   if (isManifestV3) {
     /*
      * In case of MV3 the issue of blank screen was very frequent, it is caused by UI initialising before background is ready to send state.
-     * Code below ensures that UI is rendered only after CONNECTION_READY message is received thus background is ready.
+     * Code below ensures that UI is rendered only after startUISync message is received thus background is ready.
```

Comment:
UI seems to be not only rendered after startUISync, it is also rendered on "CONNECTION_READY". What if we update the comment to something like: 
```suggestion
     * Code below ensures that UI is rendered only after "CONNECTION_READY" or "startUISync"
     * messages are received thus the background is ready, and ensures that streams and
     * phishing warning page load only after the "startUISync" message is received.
```

I left out what "startUISync" implies above because it is explained is another part of the code